src/otx/algorithms/visual_prompting/adapters/openvino/model_wrappers/openvino_models.py

```diff
@@ -94,27 +113,30 @@ def parameters(cls):  # noqa: D102
         return parameters
 
     def _get_outputs(self):
-        return "low_res_masks"
+        return "upscaled_masks"
 
     def preprocess(self, inputs: Dict[str, Any], meta: Dict[str, Any]) -> List[Dict[str, Any]]:
         """Preprocess prompts."""
         processed_prompts = []
-        # TODO (sungchul): process points
-        for bbox, label in zip(inputs["bboxes"], inputs["labels"]):
-            # TODO (sungchul): add condition to check whether using bbox or point
-            point_coords = self._apply_coords(bbox.reshape(-1, 2, 2), inputs["original_size"])
-            point_labels = np.array([2, 3], dtype=np.float32).reshape((-1, 2))
-            processed_prompts.append(
-                {
-                    "point_coords": point_coords,
-                    "point_labels": point_labels,
-                    # TODO (sungchul): how to generate mask_input and has_mask_input
-                    "mask_input": np.zeros((1, 1, 256, 256), dtype=np.float32),
-                    "has_mask_input": np.zeros((1, 1), dtype=np.float32),
-                    "orig_size": np.array(inputs["original_size"], dtype=np.float32).reshape((-1, 2)),
-                    "label": label,
-                }
-            )
+        for prompt_name in ["bboxes", "points"]:
+            for prompt, label in zip(inputs.get(prompt_name), inputs["labels"].get(prompt_name, [])):
+                if prompt_name == "bboxes":
+                    point_coords = self._apply_coords(prompt.reshape(-1, 2, 2), inputs["original_size"])
+                    point_labels = np.array([2, 3], dtype=np.float32).reshape(-1, 2)
```

Comment:
I have two questions.
Why 2x3 for point labels?
Should dtype be np.float32 for labels?

Reply:
1. `point_labels` for bboxes should have both components, 2 is top-left and 3 is bottom-right.
2. Referred to https://github.com/facebookresearch/segment-anything/blob/main/notebooks/onnx_model_example.ipynb. And I tried to use float32 for variables and int64 for size and index due to stability.